components/provisioner/e2e_test/requests/provision.graphql

```diff
@@ -0,0 +1,34 @@
+mutation ($name: String!, $provider: String!, $provider_secret: String!) {
+	provisionRuntime(
+		config: {
+			runtimeInput: { name: $name, description: "" }
+			clusterConfig: {
+				gardenerConfig: {
+					name: $name
+					kubernetesVersion: "1.26.5"
```

Comment:
Maybe we can consider passing the k8s version as a variable? With this approach, we can parameterize testing different k8s versions in the future. But it is up to you.